go/Makefile

```diff
@@ -5,6 +5,10 @@ OUT_DIR := .out
 
 all: fix fmt test test-e2e lint
 
+api:
+	(which swagger || go install github.com/go-swagger/go-swagger/cmd/swagger@v0.27.0)
+	$(GOPATH)/bin/swagger generate spec -m -w pkg/api/kptfile/v1 -o ../kptfile.yaml
```

Comment:
Can you output it to `/openapi/kptfile.yaml`?

Reply:
Done.

Reply:
Done.